docs/source/examples/pincell.ipynb

```diff
@@ -0,0 +1,1666 @@
+      "        <nuclide ao=\"0.1738\" name=\"Zr94\" />\r\n",
+      "        <nuclide ao=\"0.028\" name=\"Zr96\" />\r\n",
+      "    </material>\r\n",
+      "    <material id=\"3\" name=\"h2o\">\r\n",
+      "        <density units=\"g/cm3\" value=\"1.0\" />\r\n",
+      "        <nuclide ao=\"2.0\" name=\"H1\" />\r\n",
+      "        <nuclide ao=\"1.0\" name=\"O16\" />\r\n",
+      "        <sab name=\"c_H_in_H2O\" />\r\n",
+      "    </material>\r\n",
+      "</materials>\r\n"
+     ]
+    }
+   ],
+   "source": [
+    "mats.export_to_xml()\n",
+    "!cat materials.xml"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Element Expansion\n",
+    "\n",
+    "Did you notice something really cool that happened to our Zr element? OpenMC automatically turned it into a list of nuclides when it exported it! The way this feature works is as follows:\n",
+    "\n",
+    "- First, it checks whether `Materials.cross_sections` has been set, indicating the path to a `cross_sections.xml` file.\n",
+    "- If `Materials.cross_sections` isn't set, it looks for the `OPENMC_CROSS_SECTIONS` environment variable.\n",
+    "- If either of these are found, it scans the file to see what nuclides are actually available and will expand elements accordingly.\n",
+    "\n",
+    "Let's see what happens if we change O16 in water to elemental O."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "<?xml version='1.0' encoding='utf-8'?>\r\n",
+      "<materials>\r\n",
+      "    <material id=\"1\" name=\"uo2\">\r\n",
+      "        <density units=\"g/cm3\" value=\"10.0\" />\r\n",
+      "        <nuclide ao=\"0.03\" name=\"U235\" />\r\n",
+      "        <nuclide ao=\"0.97\" name=\"U238\" />\r\n",
+      "        <nuclide ao=\"2.0\" name=\"O16\" />\r\n",
+      "    </material>\r\n",
+      "    <material id=\"2\" name=\"zirconium\">\r\n",
+      "        <density units=\"g/cm3\" value=\"6.6\" />\r\n",
+      "        <nuclide ao=\"0.5145\" name=\"Zr90\" />\r\n",
+      "        <nuclide ao=\"0.1122\" name=\"Zr91\" />\r\n",
+      "        <nuclide ao=\"0.1715\" name=\"Zr92\" />\r\n",
+      "        <nuclide ao=\"0.1738\" name=\"Zr94\" />\r\n",
+      "        <nuclide ao=\"0.028\" name=\"Zr96\" />\r\n",
+      "    </material>\r\n",
+      "    <material id=\"3\" name=\"h2o\">\r\n",
+      "        <density units=\"g/cm3\" value=\"1.0\" />\r\n",
+      "        <nuclide ao=\"2.0\" name=\"H1\" />\r\n",
+      "        <nuclide ao=\"0.999621\" name=\"O16\" />\r\n",
+      "        <nuclide ao=\"0.000379\" name=\"O17\" />\r\n",
+      "        <sab name=\"c_H_in_H2O\" />\r\n",
+      "    </material>\r\n",
+      "</materials>\r\n"
+     ]
+    }
+   ],
+   "source": [
+    "water.remove_nuclide(openmc.Nuclide('O16'))\n",
```

Comment:
Until recently you couldn't (after I was originally creating this notebook) -- I'll change this to use just the string.